src/composer.json

```diff
@@ -24,7 +24,7 @@
         "mockery/mockery": "^1.4.4",
         "nunomaduro/collision": "^7.0",
         "nunomaduro/larastan": "^2.0",
-        "phpunit/phpunit": "^10.0",
+        "phpunit/phpunit": "10.5.31",
```

Comment:
We need to lock this until we remove php 8.1 support and move to phpunit 11. issue with paratest
See 
https://github.com/sebastianbergmann/phpunit/issues/5939 
https://github.com/sebastianbergmann/phpunit/issues/5939